release.json

```diff
@@ -4,7 +4,7 @@
   "version-upgrade-hook": "1.0.7",
   "readiness-probe": "1.0.15",
   "mongodb-agent": {
-    "version": "12.0.24.7719-1",
-    "tools_version": "100.7.0"
+    "version": "12.0.25.7724-1",
```

Comment:
LGTM! Does this need an entry in the release notes for updated images? Or should that be done when we release the operator?

Reply:
We do this at release time. But thanks for noticing it!